README.md

```diff
@@ -79,14 +79,14 @@ A ```Point``` is just a two-element array with an ```[x, y]``` coordinate.
 ```position``` is your car's current ```[x, y]``` coordinate on the racetrack.
 Units here are in grid coordinates (not pixels).
 
-```velocity``` is is your car's current volocity.  Note that the units are
+```velocity``` is is your car's current velocity.  Note that the units are
```

Comment:
I'm a big fan of https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker 😁